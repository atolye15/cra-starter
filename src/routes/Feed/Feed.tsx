import React, { FunctionComponent } from 'react';
import { Link } from 'react-router-dom';

const Feed: FunctionComponent = () => (
  <div>
    <h1>Feed</h1>

    <Link to="/">Go to home</Link>
  </div>
);

export default Feed;
