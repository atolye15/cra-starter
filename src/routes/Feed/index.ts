import Loadable from 'react-loadable';

import Loading from '../../components/Loading';

const LoadableFeed = Loadable({
  loader: () => import('./Feed'),
  loading: Loading,
});

export default LoadableFeed;
