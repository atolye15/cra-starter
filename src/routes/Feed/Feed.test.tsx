import React from 'react';
import { shallow } from 'enzyme';

import Feed from './Feed';

it('renders correctly', () => {
  const wrapper = shallow(<Feed />);

  expect(wrapper).toMatchSnapshot();
});
