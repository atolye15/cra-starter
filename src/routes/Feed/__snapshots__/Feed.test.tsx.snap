// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Feed
  </h1>
  <Link
    to="/"
  >
    Go to home
  </Link>
</div>
`;
